src/components/DetailsMetrics/ApplicationMetricCard.js

```diff
@@ -90,7 +90,7 @@ const ApplicationMetricCard = ({ metric }) => {
           </Tooltip>
         )}
       </StatsCard.Header>
-      <div className="metrics__card-body">
+      <div className="metrics__card-body metrics__card_ratio-metric">
```

Comment:
don't add classes for specific css properties. You already have the `metrics__card-metric` class. Also you can add another one for invocation chart `metrics__card-invocation`

Reply:
@mavdryk refactor CSS classes with metrics__card-metric and metrics__card-invocation